### DISCUSSION TITLE
tailwind css 3 cdn

### DISCUSSION THREAD

**Discussion by DGCodeIdeas** [tailwindlabs/tailwindcss#7345] [Help]
Please i have searched the web, i haven't found for v3.
The Play CDN is not suitable for production.
Pls i need help.
Thanks.

**Comment by HapLifeMan** [comment]
I just faced this issue today too, I wanted to update the 1.4.4 CDN to the latest version, but unfortunately I'm not able to fin any CDN having a pure CSS file. How can we achieve this?

I tried using the Play CDN but it doesn't suit for me as I don't want to have any JS on my page (and doesn't work too because I'm using puppeteer to take a screenshot of a page, mostly 75% of the time the JS didn't have the time to exec and the page is randomly CSSed) .
